internal/alb/tg/targetgroup.go

```diff
@@ -97,20 +101,21 @@ func (controller *defaultController) Reconcile(ctx context.Context, ingress *ext
 	if err = controller.targetsController.Reconcile(ctx, tgTargets); err != nil {
 		return TargetGroup{}, fmt.Errorf("failed to reconcile targetGroup targets due to %v", err)
 	}
+
 	return TargetGroup{
 		Arn:        tgArn,
 		TargetType: targetType,
 		Targets:    tgTargets.Targets,
 	}, nil
 }
 
-func (controller *defaultController) newTGInstance(ctx context.Context, name string, serviceAnnos *annotations.Service) (*elbv2.TargetGroup, error) {
+func (controller *defaultController) newTGInstance(ctx context.Context, name string, serviceAnnos *annotations.Service, healthCheckPort *string) (*elbv2.TargetGroup, error) {
```

Comment:
better to use string as argument, and pass to underline api with aws.String(healthCheckPort)